### PR TITLE
enhance: Append create partition msg to wal

### PR DESCRIPTION
**Pull request by bigsheeper**
issue: https://github.com/milvus-io/milvus/issues/33285

### CODE REVIEW

**Comment by bigsheeper** [comment]
rerun ut

**Comment by chyezh** [comment]
\lgtm

File: internal/distributed/streaming/streaming.go [hunk 1]
```diff
@@ -67,4 +61,7 @@ type WALAccesser interface {
 
 	// Read returns a scanner for reading records from the wal.
 	Read(ctx context.Context, opts ReadOption) Scanner
+
+	// Close closes the wal accesser
+	Close()
```
Comment (chyezh) [inline internal/distributed/streaming/streaming.go:66]: Close method should not be exposed.
Always use a singleton's Release operation

**Pull request merged**